packages/manager/apps/pci-project/src/data/hooks/useCart.tsx

```diff
@@ -0,0 +1,13 @@
+import { useQuery } from '@tanstack/react-query';
+import { getCartSummary } from '@/data/api/cart';
+
+export const getCartSummaryQueryKey = (cartId?: string) => [
+  `/order/cart/${cartId}/summary`,
+];
+
+export const useGetCartSummary = (cartId?: string) =>
+  useQuery({
+    queryKey: getCartSummaryQueryKey(cartId),
+    queryFn: () => getCartSummary(`${cartId}`),
```

Comment:
As `enable` protects the call, you can safely do : 


```suggestion
    queryFn: () => getCartSummary(cartId as string),
```